collect_app/src/main/java/org/odk/collect/android/utilities/TextUtils.java

```diff
@@ -69,20 +69,31 @@ private TextUtils() {
 
     }
 
-    protected static String markdownToHtml(String text) {
+    public static String markdownToHtml(String text) {
 
         text = text.replaceAll("<([^a-zA-Z/])", "&lt;$1");
         // https://github.com/enketo/enketo-transformer/blob/master/src/markdown.js
 
         // span - replaced &lt; and &gt; with <>
         text = ReplaceCallback.replace("(?s)<\\s?span([^\\/\n]*)>((?:(?!<\\/).)+)<\\/\\s?span\\s?>",
                 text, createSpan);
+
+        //intermediary replacements keys for special characters, N/B: These symbols are not meant to be interpreted as markdown
+        text = text.replaceAll("(?s)\\\\#", "&#61;");
```

Comment:
You can go straight to `&#35;` for `#`, `&#92;` for `\`, `&#95;` for `_` and `&#42;` for `*`. Otherwise you are making the intermediary characters into special characters.

Once you make those immediate changes, you should not need the two new regex for emphasis below. Then you can change back the change to `markDownToHtml_EscapesLessThan()` and change `markdownToHtml` back to `protected` and I think we'll be ready to go!

Reply:
I dont understand this part of your comment Then you can change back the change to markDownToHtml_EscapesLessThan()

Reply:
DONE @lognaturel